src/createHtml.jsx

```diff
@@ -162,13 +162,16 @@ export default function(options) {
             ? <GoogleTagManager id={googleTagManagerId}/>
             : null
           }
-
+          <script src="/shared/content/dist/dist/bundled.js"></script>
```

Comment:
Is this the `bundled.js` from layouts? This probably has a whole lot of stuff we don't want/need on these pages.